packages/browser-wallet/src/popup/page-layouts/MainLayout/i18n/da.ts

```diff
@@ -10,6 +10,8 @@ const t: typeof en = {
         accounts: 'Konti',
         ids: 'ID kort',
         settings: 'Wallet indstillinger',
+        request: 'Signatur anmodning',
```

Comment:
```suggestion
        request: 'Signaturanmodning',
```
or Anmodning om signatur.

Reply:
Anmodning om signatur it is.